pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarMetadataEventSynchronizer.java

```diff
@@ -74,10 +92,11 @@ public void start() throws PulsarServerException {
             log.info("metadata topic doesn't exist.. skipping metadata synchronizer init..");
             return;
         }
+        log.info("Metadata event synchronizer is starting on topic {}", topicName);
         this.client = (PulsarClientImpl) pulsar.getClient();
-        startProducer();
-        startConsumer();
```

Comment:
Does the consumer get started?

Reply:
Yes, it will be started after the producer is started successfully. See https://github.com/apache/pulsar/pull/22684/files#diff-a52965f323d04afed28cf4c57af8002f91fcdefe93ea90ca2fd7798a64e94636R153